android-app/app/src/main/java/de/dhbw/ka/se/fibo/utils/ApiUtils.java

```diff
@@ -23,7 +21,7 @@ private static String getBaseURL() {
         if (ActivityUtils.isEspressoTesting()) {
             return "http://localhost:8000";
         } else {
-            return "http://10.0.2.2:8000";
+            return "https://fibo.gtzcloud.de";
```

Comment:
In our Emulator, we should still use 10.0.2.2. Otherwise, we will always call your server, no matter how our local database looks like.
Only when in production we should use this URL. You could use the BuildConfig.DEBUG to determine whether we are in production (real device) or in debug (the emulator)